sign/sign.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package sign
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"github.com/agl/ed25519"
+	"github.com/keybase/client/go/saltpack"
+	"github.com/keybase/client/go/saltpack/basic"
```

Comment:
i think we should be using the `libkb/saltpack_*` keyring here, and not the `saltpack/basic` one. Can this repo depend on `libkb`?